libraries/bsg_manycore.cpp

```diff
@@ -254,6 +258,21 @@ static int hb_mc_manycore_rx_fifo_drain(hb_mc_manycore_t *mc, hb_mc_fifo_rx_t ty
 // Init/Exit API //
 ///////////////////
 
+#ifdef COSIM
+static int hb_mc_manycore_init_dpi(hb_mc_manycore_t *mc)
+{
+        svScope scope;
+        scope = svGetScopeFromName("tb");
+        svSetScope(scope);
+        return HB_MC_SUCCESS;
+}
+
+static void hb_mc_manycore_cleanup_dpi(hb_mc_manycore_t *mc)
+{
+        return;
```

Comment:
What's the job of the function? Are we going to do something in this later?

Reply:
Yep. For an incoming feature :p 

(All of my current PRs are for a fancy new feature)